styles.css

```diff
@@ -395,3 +229,264 @@ body {
+  }
+  
+  .local {
+    color: #E07A5F;
+  }
+  
+  .nav-right-side {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding-right: 2%;
+  }
+
+  .user-drop-down {
+    font-size: .65em;
+    font-weight: 200;
+    width: 80%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+  }
+  
+  .title {
+    font-size: 1.6em;
+    text-align: right;
+    margin-right: 5%;
+    letter-spacing: .1em;
+  }
+  
+  .icon {
+    width: 2em;
+    height: 2em;
+  }
+  
+  .mushroom-icon {
+    width: 1em;
+    height: 1em;
+  }
+  
+  .main {
+    grid-area: main;
+    height: 100%;
+    width: 100%;
+  }
+  
+  .page-heading {
+    letter-spacing: .05em;
+    margin-left: 3%;
+    color: #585959;
+    padding-top: 2%;
+    height: 3%;
+  }
+  
+  .mushroom-cards {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    gap: 5%;
+    column-gap: 0;
+    padding: 3% 0 5% 3%;
+    height: 100%;
+    width: 100%;
+  }
+  
+  .card {
+    background: #CFDAE1;
+    box-shadow: 0 0 6px 6px #C1C1C1;
+    width: 80%;
+    height: 100%;
+    margin-bottom: 10%;
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 35% 65%;
+    grid-template-areas:
+      "image"
+      "content";
+  }
+  
+  .card-top {
+    grid-area: image;
+    position: relative;
+  }
+  
+  .card-body {
+    height: 100%;
+    width: 100%;
+  }
+  
+  .mushroom-pic {
+    width: 100%;
+    height: 100%;
+  }
+  
+  .found-label {
+    background-color: #E56635;
+    color: #EFEFEF;
+    width: 25%;
+    height: 35%; 
+    border-radius: 50%;
+    position: absolute;
+    top: 10%;
+    right: 3%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+  
+  .plus-icon {
+    font-size: 4em;
+    font-weight: 500;
+    color: #EFEFEF;
+    position: absolute;
+    top: -30%;
+    left: 1%;
+  }
+  
+  .found {
+    font-size: 1em;
+    font-weight: 200;
+    margin: 0;
+  }
+  
+  .number-sightings {
+    margin: 0;
+    font-size: 1em;
+  }
+  
+  .card-body {
+    grid-area: content;
+    margin-left: 5%;
+    color: #585959;
+  }
+  
+  .mushroom-name {
+    margin: 5% 0 0 0;
+    font-size: 1.6em;
+  }
+  
+  .scientific-name {
+    margin: 0 0 0 0;
+    font-style: italic;
+    font-size: 1.2em;
+  }
+
+  .characteristics {
+    background-color: #EFEFEF;
+    width: 84%;
+    height: 16%; 
+    padding: 3%; 
+    font-size: 1em;
+  }
+  
+  .season-header {
+    margin: 5% 0 0 0;
+  }
+  
+  .seasons {
+    margin: 1% 0 0 0;
+    font-size: 1.6em;
+  }
+  
+  .rare-header {
+    margin: 5% 0 0 0;
+  }
+  
+  .rarity {
+    margin: 2% 0 0 0;
+    font-size: 2em;
+    color: #E56635;
+    font-weight: 500;
+  }
+
+
+}
+
+/* DESKTOP DISPLAY */
+
+@media all and (min-width: 1200px) {
+  
+  .nav-list {
+    flex-direction: row;
+    align-items: flex-end;
+    padding-bottom: 2%;
+    font-size: 1.2em;
+    width: 45%;
```

Comment:
One way to address the nav items' text wrapping would be to make this list a bit bigger on desktop.